CMakeLists.txt

```diff
@@ -1088,6 +1088,8 @@ if(NOT MSVC)
   append_cxx_flag_if_supported("-fno-math-errno" CMAKE_CXX_FLAGS)
   append_cxx_flag_if_supported("-fno-trapping-math" CMAKE_CXX_FLAGS)
   append_cxx_flag_if_supported("-Werror=format" CMAKE_CXX_FLAGS)
+  append_cxx_flag_if_supported("-Wno-error=dangling-reference" CMAKE_CXX_FLAGS)
+  append_cxx_flag_if_supported("-Wno-error=redundant-move" CMAKE_CXX_FLAGS)
```

Comment:
Let's restrict those to a specific specific GCC versions and if the are arechitecture specific let's restrict them to s360